docs/source/06_nodes_and_pipelines/03_modular_pipelines.md

```diff
@@ -4,406 +4,267 @@
+### What does the ``kedro pipeline create`` do?
 
-Kedro will also parse any requirements packaged with the modular pipeline and add them to project level `requirements.in`. It is advised to do `kedro install --build-reqs` to compile and install the updated list of requirements after pulling a modular pipeline.
+Running the pipeline create command adds the following folder structure to your project. For your convenience Kedro gives you a pipeline specific `nodes.py`, `pipeline.py`, parameters and appropriate `tests` structure. You also don't have to add those pesky `__init__.py` files yourself, which is handy 😅.
 
-```eval_rst
-.. note::  If a modular pipeline has embedded requirements and a project `requirements.in` file does not already exist, it will be generated based on the project `requirements.txt` before appending the modular pipeline requirements.
+<details>
+<summary><b>Click to see the generated folder structure</b></summary>
+
+```text
+├── conf
+│   └── base
+│       └── parameters
+│           └── {{pipeline_name}}.yml  <-- Pipeline specific parameters
+└── src
+    ├── my_project
+    │   ├── __init__.py
+    │   └── pipelines
+    │       ├── __init__.py
+    │       └── {{pipeline_name}}      <-- This folder defines the modular pipeline
+    │           ├── README.md          <-- To store pipeline specific documentation 
+    │           ├── __init__.py        <-- So that Python treats this pipeline as a module
+    │           ├── nodes.py           <-- To declare your nodes
+    │           └── pipeline.py        <-- To structure the pipeline itself
+    ├── tests
+    │   ├── __init__.py
+    │   └── pipelines
+    │       ├── __init__.py
+    │       └── {{pipeline_name}}      <-- Pipeline specific tests
+    │           ├── __init__.py
+    │           └── test_pipeline.py
+    └──  pipeline_registry.py          <-- Does NOT automatically update the registry 
 ```
 
-You can pull a modular pipeline from different locations, including local storage, PyPI and the cloud:
+</details>
 
-- Pulling a modular pipeline from a local directory:
+If you want to do the reverse and remove a modular pipeline, you can use ``kedro pipeline delete`` to do so.
 
-```bash
-kedro pipeline pull <path-to-your-project-root>/src/dist/<pipeline_name>-0.1-py3-none-any.whl
-```
+### Ensuring portability
 
-- Pulling a modular pipeline from S3:
+Kedro modular pipelines are shareable between Kedro codebases via [micro-packaging](04_micro_packaging.md), but you need to follow a couple of rules to ensure portability:
 
-```bash
-kedro pipeline pull https://<bucket_name>.s3.<aws-region>.amazonaws.com/<pipeline_name>-0.1-py3-none-any.whl
-```
-
-- Pulling a modular pipeline from PyPI:
+* Modular pipelines should use [relative imports](https://docs.python.org/3/reference/import.html#package-relative-imports) (as opposed to 'absolute' imports) when accessing their own objects, but are free to use absolute imports otherwise.
```

Comment:
I didn't think that was still true since using `rope`? Unless I'm misreading the sentence.

Reply:
I just took it from the old stuff so will remove.
```suggestion
```